src/generator/renderer.clj

```diff
@@ -264,13 +269,40 @@
       [:h2 (get-in contentful-map [:articleList :websiteTitle])]]
      (into [:ul.post-list] (mapv render (get-in contentful-map [:articleList :linkedFrom :postCollection :items])))]))
 
-; Todo implement two person logics: embedded and person page.
-(defmethod render "Person"
+(defmethod render "PeopleList"
+  [args]
+  (let [contentful-map (contentful/get-contentful
+                        :people-by-list-query {:listId (get-in args [:sys :id])})]
+    [:section.people-list-wrap 
+     (into [:ul.people-list] (mapv render (get-in contentful-map [:peopleList :linkedFrom :personCollection :items])))]))
+
+;;person-singleview and people-list view repeat to make styling and content changes easier
+(defn person-single-view
   [args]
   [:div.person
-   [:span.name (:name args)]])
+   [:a {:href (create-url (:slug args))} [:h2 (:name args)]]
+   [:div.image
+    [:img {:alt (get-in args [:picture :title]) :src (:url (:picture args))}]]
+   [:div.body
+    [:h4 (:jobTitle args)]
+    [:p.office (:name (:office args))]
+    [:div.person-body (richtext->html (:json (:description args)))]]])
+
+(defn people-list-view
+  [args]
+  [:li.person-body
+   [:a {:href (create-url (:slug args))} [:h3 (:name args)]]
+   [:div.image
+    [:img {:alt (get-in args [:picture :title]) :src (:url (:picture args))}]]
+   [:div.body
+    [:h4 (:jobTitle args)]
+    [:div.person-body (richtext->html (:json (:shortText args)))]]])
+
+(defmethod render "Person"
```

Comment:
Note about code organization and readability. 

Group "defmethod" calls together so that the code functions remain scannable.
Having defn calls between the def method calls causes difficulties in scanning the render method. while trying to answer questions like "What methods are available for 'defmethod' call?

you can move the get single & get person above the entire "render" method.